hawc/apps/assessment/api.py

```diff
@@ -52,6 +52,30 @@ def get_assessment_from_query(request) -> Optional[models.Assessment]:
     return models.Assessment.objects.filter(pk=assessment_id).first()
 
 
+class AssessmentForeignKeyPermissions(permissions.BasePermission):
```

Comment:
let's add a comment to describe what this is doing. I'd probably call it JobPermissions or something b/c it's really specialized to this use case at this point